README.md

```diff
@@ -39,11 +37,15 @@ Install the linters folder to your project.
 Install the fontawesome for icons in the project (Preferrable)
 
 
+
 ### Usage
 
 ### Run tests
 
 
+### Deployment
+The project  was deployed on Github pages and can be viewed on the following link: https://marlyn-mayienga.github.io/Marlyn_Mayienga/
```

Comment:
## OPTIONAL 

- [ ] Kindly use this format for links in your readme `[live demo](https://marlyn-mayienga.github.io/Marlyn_Mayienga/)`